src/Features/Core/Portable/EncapsulateField/AbstractEncapsulateFieldService.cs

```diff
@@ -358,8 +360,9 @@ protected string GeneratePropertyName(string fieldName)
                 baseName = fieldName;
             }
 
-            // Make uppercase the first letter
-            return char.ToUpper(baseName[0]).ToString() + baseName.Substring(1);
+            // Make the first character upper case using the "en-US" culture
+            var firstCharacter = _englishUSCulture.TextInfo.ToUpper(baseName[0]);
```

Comment:
The portable APIs don't have `char.ToUpper(char, CultureInfo)` but according to the [reference source](http://referencesource.microsoft.com/#mscorlib/system/char.cs,8a74a9e898d17637) for that method, I can get the same result by using `TextInfo`.